takeNotes-src/postNotes/app.js

```diff
@@ -0,0 +1,89 @@
+// default imports
+const AWS = require("aws-sdk");
+const DDB = new AWS.DynamoDB({ apiVersion: "2012-10-08" });
+const { v4: uuidv4 } = require("uuid");
+
+// environment variables
+const { TABLE_NAME, ENDPOINT_OVERRIDE, REGION } = process.env;
+const options = { region: REGION };
+AWS.config.update({ region: REGION });
+
+if (ENDPOINT_OVERRIDE !== "") {
+  options.endpoint = ENDPOINT_OVERRIDE;
+}
+
+const docClient = new AWS.DynamoDB.DocumentClient(options);
+
+// response helper
+const response = (statusCode, body, additionalHeaders) => ({
+  statusCode,
+  body: JSON.stringify(body),
+  headers: {
+    "Content-Type": "application/json",
+    "Access-Control-Allow-Origin": "*",
+    ...additionalHeaders,
+  },
+});
+
+function isValidRequest(context, event) {
+  let isIdValid =
+    event !== null &&
+    event.pathParameters !== null &&
+    event.pathParameters.id !== null;
+
+  let body = event.body;
+  let isBodyValid = body !== null && body.notes !== null;
+
+  return isIdValid && isBodyValid;
+}
+
+function updateRecord(recordId, eventBody, id) {
+  let d = new Date();
+  console.log("record id: " + recordId + " eventBody: " + eventBody.notes);
+  let noteBody = eventBody.notes;
```

Comment:
should we rename this to `note`? From what I see its just one note right?

Reply:
yes good catch